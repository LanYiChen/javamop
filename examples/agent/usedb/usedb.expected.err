Downloading specs from https://github.com/runtimeverification/property-db.git ...
Done downloading specs.
Copying Specs ...
