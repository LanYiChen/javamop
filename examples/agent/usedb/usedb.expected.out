-Processing 181 specification(s)
 Reader_UnmarkedReset.rvm is generated
 Writer_ManipulateAfterClose.rvm is generated
 PushbackInputStream_UnreadAheadLimit.rvm is generated
 StreamTokenizer_AccessInvalidField.rvm is generated
 InputStream_MarkReset.rvm is generated
 Closeable_MultipleClose.rvm is generated
 File_LengthOnDirectory.rvm is generated
 Closeable_MeaninglessClose.rvm is generated
 File_DeleteTempFile.rvm is generated
 ObjectOutput_Close.rvm is generated
 Console_FillZeroPassword.rvm is generated
 Console_CloseWriter.rvm is generated
 Reader_ManipulateAfterClose.rvm is generated
 InputStream_ReadAheadLimit.rvm is generated
 OutputStream_ManipulateAfterClose.rvm is generated
 RandomAccessFile_ManipulateAfterClose.rvm is generated
 InputStream_MarkAfterClose.rvm is generated
 Console_CloseReader.rvm is generated
 ObjectInput_Close.rvm is generated
 BufferedInputStream_SynchronizedFill.rvm is generated
 Reader_MarkReset.rvm is generated
 Reader_ReadAheadLimit.rvm is generated
 Serializable_NoArgConstructor.rvm is generated
 PipedStream_SingleThread.rvm is generated
 ByteArrayOutputStream_FlushBeforeRetrieve.rvm is generated
 InputStream_UnmarkedReset.rvm is generated
 InputStream_ManipulateAfterClose.rvm is generated
 Serializable_UID.rvm is generated
 PipedOutputStream_UnconnectedWrite.rvm is generated
 PipedInputStream_UnconnectedRead.rvm is generated
 ObjectStreamClass_Initialize.rvm is generated
 URLConnection_Connect.rvm is generated
 Socket_SetTimeoutBeforeBlockingInput.rvm is generated
 URLConnection_SetBeforeConnect.rvm is generated
 NetPermission_Actions.rvm is generated
 MulticastSocket_TTL.rvm is generated
 DatagramSocket_TrafficClass.rvm is generated
 URLDecoder_DecodeUTF8.rvm is generated
 Socket_Timeout.rvm is generated
 URLConnection_OverrideGetPermission.rvm is generated
 DatagramSocket_SoTimeout.rvm is generated
 Socket_ReuseAddress.rvm is generated
 ServerSocket_PerformancePreferences.rvm is generated
 ServerSocket_Timeout.rvm is generated
 DatagramSocket_Port.rvm is generated
 ContentHandler_GetContent.rvm is generated
 HttpCookie_Domain.rvm is generated
 DatagramPacket_Length.rvm is generated
 Socket_CloseInput.rvm is generated
 InetSocketAddress_Port.rvm is generated
 ServerSocket_LargeReceiveBuffer.rvm is generated
 PasswordAuthentication_FillZeroPassword.rvm is generated
 IDN_ToAscii.rvm is generated
 Socket_ReuseSocket.rvm is generated
 DatagramPacket_SetLength.rvm is generated
 Socket_TrafficClass.rvm is generated
 URLEncoder_EncodeUTF8.rvm is generated
 Socket_PerformancePreferences.rvm is generated
 Socket_LargeReceiveBuffer.rvm is generated
 ServerSocket_Backlog.rvm is generated
 Socket_InputStreamUnavailable.rvm is generated
 SocketImpl_CloseOutput.rvm is generated
 URL_SetURLStreamHandlerFactory.rvm is generated
 HttpURLConnection_SetBeforeConnect.rvm is generated
 Socket_CloseOutput.rvm is generated
 InetAddress_IsReachable.rvm is generated
 SocketPermission_Actions.rvm is generated
 ServerSocket_Port.rvm is generated
 Socket_SetTimeoutBeforeBlockingOutput.rvm is generated
 ServerSocket_ReuseAddress.rvm is generated
 HttpCookie_Name.rvm is generated
 Socket_OutputStreamUnavailable.rvm is generated
 ServerSocket_SetTimeoutBeforeBlocking.rvm is generated
 NetPermission_Name.rvm is generated
 Authenticator_OverrideGetPasswordAuthentication.rvm is generated
 List_UnsafeListIterator.rvm is generated
 Arrays_Comparable.rvm is generated
 ListIterator_RemoveOnce.rvm is generated
 Scanner_ManipulateAfterClose.rvm is generated
 SortedMap_StandardConstructors.rvm is generated
 Properties_ManipulateAfterLoad.rvm is generated
 PriorityQueue_NonComparable.rvm is generated
 TreeSet_Comparable.rvm is generated
 Vector_InsertIndex.rvm is generated
 ArrayDeque_UnsafeIterator.rvm is generated
 ArrayDeque_NonNull.rvm is generated
 NavigableSet_Modification.rvm is generated
 EnumMap_NonNull.rvm is generated
 Collection_UnsafeIterator.rvm is generated
 List_UnsynchronizedSubList.rvm is generated
 Scanner_SearchAfterClose.rvm is generated
 Collections_SynchronizedMap.rvm is generated
 Arrays_SortBeforeBinarySearch.rvm is generated
 Map_StandardConstructors.rvm is generated
 Random_OverrideNext.rvm is generated
 Collection_HashCode.rvm is generated
 Map_ItselfAsValue.rvm is generated
 Dictionary_Obsolete.rvm is generated
 Collections_CopySize.rvm is generated
 SortedSet_Comparable.rvm is generated
 Collections_UnnecessaryNewSetFromMap.rvm is generated
 Map_UnsafeIterator.rvm is generated
 ListIterator_Set.rvm is generated
 NavigableMap_Modification.rvm is generated
 Iterator_HasNext.rvm is generated
 Collections_ImplementComparable.rvm is generated
 Collections_Comparable.rvm is generated
 Collections_SortBeforeBinarySearch.rvm is generated
 PriorityQueue_NonNull.rvm is generated
 ResourceBundleControl_MutateFormatList.rvm is generated
 Collection_UnsynchronizedAddAll.rvm is generated
 Map_UnsynchronizedAddAll.rvm is generated
 ServiceLoaderIterator_Remove.rvm is generated
 TreeMap_Comparable.rvm is generated
 ServiceLoader_MultipleConcurrentThreads.rvm is generated
 Collections_SynchronizedCollection.rvm is generated
 StringTokenizer_HasMoreElements.rvm is generated
 Arrays_MutuallyComparable.rvm is generated
 NavigableMap_UnsafeIterator.rvm is generated
 Map_ItselfAsKey.rvm is generated
 Dictionary_NullKeyOrValue.rvm is generated
 ListIterator_hasNextPrevious.rvm is generated
 Map_CollectionViewAdd.rvm is generated
 Iterator_RemoveOnce.rvm is generated
 Deque_OfferRatherThanAdd.rvm is generated
 Collections_NewSetFromMap.rvm is generated
 Set_ItselfAsElement.rvm is generated
 Collection_StandardConstructors.rvm is generated
 SortedSet_StandardConstructors.rvm is generated
 EnumSet_NonNull.rvm is generated
 InvalidPropertiesFormatException_NonSerializable.rvm is generated
 Enumeration_Obsolete.rvm is generated
 Arrays_DeepHashCode.rvm is generated
 ClassLoader_UnsafeClassDefinition.rvm is generated
 ProcessBuilder_NullKeyOrValue.rvm is generated
 Enum_NoOrdinal.rvm is generated
 Float_StaticFactory.rvm is generated
 CharSequence_NotInSet.rvm is generated
 Appendable_ThreadSafe.rvm is generated
 Enum_NoExtraWhiteSpace.rvm is generated
 Byte_BadParsingArgs.rvm is generated
 StringBuffer_SingleThreadUsage.rvm is generated
 Thread_StartOnce.rvm is generated
 Long_StaticFactory.rvm is generated
 StringBuilder_ThreadSafe.rvm is generated
 ProcessBuilder_ThreadSafe.rvm is generated
 ShutdownHook_SystemExit.rvm is generated
 CharSequence_NotInMap.rvm is generated
 ShutdownHook_PrematureStart.rvm is generated
 Thread_SetDaemonBeforeStart.rvm is generated
 Character_ValidateChar.rvm is generated
 ShutdownHook_UnsafeAWTCall.rvm is generated
 Short_BadParsingArgs.rvm is generated
 Math_ContendedRandom.rvm is generated
 Byte_StaticFactory.rvm is generated
 Boolean_StaticFactory.rvm is generated
 RuntimePermission_NullAction.rvm is generated
 System_NullArrayCopy.rvm is generated
 Long_BadDecodeArg.rvm is generated
 Runnable_OverrideRun.rvm is generated
 String_UseStringBuilder.rvm is generated
 Byte_BadDecodeArg.rvm is generated
 Comparable_CompareToNull.rvm is generated
 Double_StaticFactory.rvm is generated
 Long_BadParsingArgs.rvm is generated
 ShutdownHook_UnsafeSwingCall.rvm is generated
 RuntimePermission_PermName.rvm is generated
 CharSequence_UndefinedHashCode.rvm is generated
 Enum_UserFriendlyName.rvm is generated
 Short_StaticFactory.rvm is generated
 Object_NoClone.rvm is generated
 ShutdownHook_LateRegister.rvm is generated
 SecurityManager_Permission.rvm is generated
 Character_StaticFactory.rvm is generated
 Object_MonitorOwner.rvm is generated
 Integer_BadDecodeArg.rvm is generated
 Short_BadDecodeArg.rvm is generated
 Throwable_InitCauseOnce.rvm is generated
 Integer_StaticFactory.rvm is generated
 StrictMath_ContendedRandom.rvm is generated
 System_WrongKeyOrValue.rvm is generated
 agentMonitorAspect.aj is generated
-Processing 181 specification(s)
 agentRuntimeMonitor.java is generated
agent.jar is generated.
